README.md

```diff
@@ -1,30 +1,27 @@
 ![Arrow Banner](https://github.com/Arrow-air/.github/raw/main/profile/assets/arrow_v2_twitter-banner_neu.png)
 
-# MODULE_NAME Service
+# <LIBRARY_NAME> Library
```

Comment:
Let's try to make things consistent across the org. Either wrap placeholders with angular brackets or without it. But let's not have different conventions for structurally similar contents.

Reply:
Fixed in https://github.com/Arrow-air/template-rust-lib/pull/1/commits/10f99ba487d694fce868138e25b71987138371f9